spec/support/disability_compensation_form/all_claims_fe_submission.json

```diff
@@ -152,11 +152,40 @@
         "afghanistan": true,
         "bahrain": true,
         "jordan": true,
-        "kuwait": true
+        "kuwait": true,
+        "iraq": true,
+        "qatar": true
       },
       "gulfWar2001": {
         "djibouti": true,
-        "lebanon": true
+        "lebanon": true,
+        "uzbekistan": true,
+        "airspace": true
```

Comment:
FE may also send values with `none` or `notsure`. Is this something that we should include in unit tests?

Reply:
We're already testing these scenarios [here](https://github.com/department-of-veterans-affairs/vets-api/blob/76c1cd11159e71387ff6c6f813a0e12cd8838d02/spec/lib/evss/disability_compensation_form/form526_to_lighthouse_transform_spec.rb#L293)- not explicitly with herbicide locations, but both gulf war and herbicide run through the same common logic

Reply:
awesome, thanks!